en/config/actuators.md

```diff
@@ -1,139 +1,413 @@
+  - If `Custom` is selected, then the UI displays three additional fields for setting the motor orientation.
 
-Some vehicles may use bidirectional motors (i.e. motor spins in direction 1 for lower output range and in direction 2 for the upper half).
+#### Motor Geometry: Other Vehicles
+
+Other vehicle types will define an appropriate motor geometry for their frame type.
+Once again these motors will generally have the same kinds of properties as shown above.
+
+For example, a fixed-wing vehicle may just have a single pusher moter, while a rover with differential steering will have a motor for throttle and for steering.
+
+
+#### Motor Position Coordinate System
+
+The coordinate system for motor positions is FRD (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+
+The **origin is the vehicle's centre-of-gravity (COG)**, and **NOT** the autopilot location).
+
+![Actuators CG reference diagram](../../assets/config/actuators/quadcopter_actuators_cg_reference.png)
+
+#### Bidirectional Motors
+
+Some vehicles may use bidirectional motors (i.e. motor supports spinning in both directions)
 For example, ground vehicles that want to move forwards and backwards, or VTOL vehicles that have pusher motors that go in either direction.
 
 If bidiectional motors are used, make sure to select the **Reversible** checkbox for those motors (the checkbox is displayed as an "advanced" option).
 
-![Reversible](../../assets/config/actuators/qgc_geometry_reversible.png)
+![Reversible](../../assets/config/actuators/qgc_geometry_reversible_marked.png)
 
 Note that you will need to also ensure that the ESC associated with bidirectional motors is configured appropriately (e.g. 3D mode enabled for DShot ESCs, which can be achieved via [DShot commands](../peripherals/dshot.md#commands)).
 
 
+### Control Surfaces Geometry
+
+The control surfaces section of the geometry panel lets you set the number of control surfaces that are present on the vehicle.
+You can then specify the type of each individual surface, and configure the appropriate scaling values for the Roll scale, Yaw scale, Pitch scale, Trim and Slew rate (advanced).
+
+An "example" control surface section for a vehicle with two ailerons is shown below.
+Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
+
+![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
+
+The fields are:
+
+- `Control Surfaces`: The number of control surfaces (set this first!)
+- `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
+- `Roll scale`: Roll scale (normalised: -1 to 1)
+- `Pitch scale`: Pitch scale (normalised: -1 to 1)
+- `Yaw scale`: Yaw scale (normalised: -1 to 1)
```

Comment:
@sfuhrer Thanks for your [response above](https://github.com/PX4/PX4-user_guide/pull/1858/files#r881534338) on these.  To improve this I added a section with your information about the scaling below - https://github.com/PX4/PX4-user_guide/pull/1858/files#r882271662

Then I link to this here. Is this OK?

```suggestion
- `Roll scale`: Effectiveness of actuator around roll axis (normalised: -1 to 1). [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
- `Pitch scale`: : Effectiveness of actuator around pitch axis (normalised: -1 to 1).  [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
- `Yaw scale`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).  [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
```